src/Fhir.Anonymizer.Core/Processors/EncryptProcessor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Text;
+using Fhir.Anonymizer.Core.Models;
+using Fhir.Anonymizer.Core.Utility;
+using Hl7.Fhir.ElementModel;
+using Microsoft.Extensions.Logging;
+
+namespace Fhir.Anonymizer.Core.Processors
+{
+    public class EncryptProcessor: IAnonymizerProcessor
+    {
+        private readonly byte[] _key;
+        private readonly ILogger _logger = AnonymizerLogging.CreateLogger<EncryptProcessor>();
+
+        public EncryptProcessor(string encryptKey)
+        {
+            _key = Encoding.UTF8.GetBytes(encryptKey);
+        }
+
+        public ProcessResult Process(ElementNode node)
+        {
```

Comment:
Just curious can we configure entrypt for every kinds of data types?

Reply:
Yes. We encrypt all fields according to configuration file and the author will decide the format compatibility issues by himself.

Reply:
We will also address this in the document that the encrypt method is better to process field without format constraints

Reply:
Got it. Thanks!